src/utils.ts

```diff
@@ -3,4 +3,9 @@ export default class Utils {
         const config = require(`../platforms/${platformId}/config.json`);
         return config;
     }
-}
\ No newline at end of file
+
+    public static getTemplate(platformId: string, templateId: string, type: string): any {
+        const config = require(`../platforms/${platformId}/templates/${templateId}/${type}.json`);
```

Comment:
There is no need for an additional variable here.

Reply:
Do you mean `config`?

Reply:
Yes, I meant config. Resolved that.